src/components/dropdowns/primary-action-dropdown/primary-action-dropdown.story.js

```diff
@@ -1,37 +1,35 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
-import { withKnobs, boolean, text, select } from '@storybook/addon-knobs';
+import { withKnobs, boolean, text } from '@storybook/addon-knobs';
 import withReadme from 'storybook-readme/with-readme';
 import Section from '../../../../.storybook/decorators/section';
-import * as icons from '../../icons';
+import { BoxIcon, BrainIcon, FlameIcon } from '../../icons';
 import Readme from './README.md';
 import PrimaryActionDropdown, { Option } from './primary-action-dropdown';
 
-const iconNames = Object.keys(icons);
-
 storiesOf('Components|Dropdowns', module)
   .addDecorator(withKnobs)
   .addDecorator(withReadme(Readme))
   .add('PrimaryActionDropdown', () => (
     <Section>
       <PrimaryActionDropdown>
         <Option
-          iconLeft={React.createElement(
-            icons[select('icon', iconNames, iconNames[0])]
-          )}
+          iconLeft={<BoxIcon />}
```

Comment:
It was failing because only the first option had an icon. When the first option didn't have an icon, the `React.cloneElement` done inside of `primary-action-dropdown.js` failed.

Back when this component was created, we decided that every option needs to have an icon. This PR ensures there is an icon for every option.

I removed the possibility to select the icon to keep the story simple.